app/views/groups/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<% if user_signed_in? %>
+  <header class="container-fluid bg-primary py-2 px-3">
+    <nav class="d-flex w-100 justify-content-between align-items-center">
+      <%= link_to "Logout", destroy_user_session_path, method: :delete, class: 'text-white text-decoration-none nova-bold'  %>
+      <h6 class="text-white m-0 nova-bold">CATEGORIES</h6>
+      <%= link_to "Add", new_group_path, method: :get, class: 'text-white text-decoration-none nova-bold'  %>
+    </nav>
+  </header>
+  <section class="container-fluid vh-100 w-100 bg-light">
+    <%= render "flash_notification" %>
+    <div class="row p-3">
+      <ul class='m-0 p-0'>
+        <% @groups.each do |group| %>
+          <%= link_to(group_path(group), class: 'text-decoration-none') do %>
+            <li class="d-flex justify-content-evenly py-3 bg-white mb-3 align-items-center">
+              <div class="w-25 text-center">
+                <img src=<%= group.icon %> width='20' />
+              </div>
+              <h6 class="m-0 w-25 text-center"><%= group.name.capitalize %></h6>
+              <h6 class="m-0 w-25 text-center nova-bold">Total: $<%= group.group_contracts.sum { |gc| gc.contract.amount } %></h6>
+            </li>
+          <% end %>
+        <% end %>
+      </ul>
```

Comment:
- [ ] You did an amazing work overall :clap: :clap: . The app works perfectly. But there are a few issues with the design requirements. You need to strictly follow the given design. The category components here are not as visible as the original design. Kindly improve the font size, element spacing, padding, etc... properties so that your app's UI looks like the original design. I have checked your app in mobile view and it is not as user friendly as the original design. Kindly make changes. :pray: 

![image](https://user-images.githubusercontent.com/64894927/142999139-ed435168-aa50-44f2-a9ea-fb7f4b6a6d87.png)

![image](https://user-images.githubusercontent.com/64894927/142999195-a0d8a90e-3071-4c33-a843-ae3e3a6069fe.png)

Reply:
Thanks for the review. I did the changes an now looks better.

  <img width="408" alt="imagen" src="https://user-images.githubusercontent.com/39852288/143025440-c3c05483-2351-4f13-b758-b8da35ba5082.png">